### DISCUSSION TITLE
macOS 15 issue/support?

### DISCUSSION THREAD

**Discussion by rflois** [openscopeproject/TrguiNG#230] [Q&A] [answered]
I just upgraded to macOS 15 Sequoia. Can't say this is what is causing the issue, though it does seem that all applications that need local network access now require (new?)  approval in System Settings. Doesn't look like you can manually add an application, and TrguiNG doesn't show up in the list. I tried creating a new server within the app, still doesn't connect. Confirmed that Transmission is running normally via the browser and within the Docker container it's always been running in.

Anyone else running into this?

**Comment by rflois** [comment, marked as answer]
Confirmed this is the issue. Redownloaded and installed the app.  Had to manually allow it to run within Privacy and Security, and then, when it finally launched for the first time, it asked for local network permission.  Looks like a full manual reinstall is necessary. Not sure of a better way to handle this.

**Reply by qu1ck** [reply]
That's probably the best way to handle this for the foreseeable future. I don't have a modern mac (or any mac for that matter, only a VM which doesn't support modern macos) so I won't be able to fix or test this.

**Reply by rflois** [reply]
If you need any help with testing things along the way, please let me know :). Really appreciate all of the work you've done here!
